### DISCUSSION TITLE
Moving settings to another computer

### DISCUSSION THREAD

**Discussion by gerroon** [kingToolbox/WindTerm#635] [Q&A] [answered]
Hi

I need to move from one Win Pc to another one. What is the right way to move my WT settings so that I have all the sessions etc on that computer?

**Comment by kingToolbox** [comment, marked as answer]
Hello, the steps have been described in detail on the release page, and you can follow the steps:

**Versions before Version 2.4.0**
- Start WindTerm and select the profiles directory and quit.
- Replace the new .wind/profiles folder with the old profiles folder.
- Restart WindTerm and all the sessions and configuration will be restored.

**Versions after Version 2.4.0**
- Start WindTerm and select the profiles directory containing the old .wind folder.
- It is recommended to choose the Home Directory to facilitate subsequent upgrades.

**If the .wind folder needs to be moved to another system:**
- Quit WindTerm.
- Replace the new .wind folder with the old .wind folder.
- Restart WindTerm and all the sessions and configuration will be restored.

The above description is about the `WindTerm_2.4.0_Prerelease_1` version. Please confirm your version first. Thank you.
